### PR TITLE
(MCO-726) Prepare for 2.8.6 release

### PR DESCRIPTION
**Pull request by shrug**
Bump the version and update the changelog and readme for 2.8.6 release

### CODE REVIEW

File: website/changelog.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ toc: false
 
 |Date|Description|Ticket|
 |----|-----------|------|
+|2015/09/15|Release *2.8.6*|MCO-726|
+|2015/09/11|Fix solaris smf service manifest|(#345)[https://github.com/puppetlabs/marionette-collective/pull/345]|
```
Comment (melissa) [inline website/changelog.md]: Wasn't there also an MCO ticket for this change?

Comment (shrug) [inline website/changelog.md]: Was there? it wasn't noted in the PR or commit message

Comment (melissa) [inline website/changelog.md]: Ah, since it's an RE ticket, let's leave it out

File: website/releasenotes.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ This is a list of release notes for various releases, you should review these
...
 will be highlighted here.
 
+
+<a name="2_8_6">&nbsp;</a>
+
+## 2.8.6 - 2015/09/15
+
+### Changes since 2.8.5
+
+* Fixed an issue with the solaris smf service
```
Comment (melissa) [inline website/releasenotes.md]: I feel like a little more information would be nice to have

**Comment by melissa** [comment]
:+1: I'd really like to get someone from mco to take a look at this too, if that's possible

**Comment by kylog** [comment]
:+1:

**Pull request merged**